Adapt MediaRecorder API to new events structure (#13850)
mdn/content: files/en-us/web/api/mediarecorder/resume_event/index.md

>   - Property

Reviewer: ```suggestion
  - Event
```
Or it won't be listed correctly in the sidebar.

  - Reference
  - Video
  - resume
browser-compat: api.MediaRecorder.resume_event
---
{{APIRef("Media Recorder API")}}

> The `resume` event is thrown as a result of the
> {{domxref("MediaRecorder.resume()")}} method being invoked.

Reviewer 2: ```suggestion
The `resume` event is thrown when
{{domxref("MediaRecorder.resume()")}} is called.
```
